src/c++/perf_analyzer/test_command_line_parser.cc

```diff
@@ -1173,6 +1176,62 @@ TEST_CASE("Testing Command Line Parser")
 
       check_params = false;
     }
+
+    SUBCASE("Max threads set to default when concurrency-range < 16")
+    {
+      args.push_back(option_name);
+      args.push_back("10:10");  // start
+
+      int argc = args.size();
+      char* argv[argc];
+      std::copy(args.begin(), args.end(), argv);
+
+      REQUIRE_NOTHROW(act = parser.Parse(argc, argv));
+      CHECK(!parser.UsageCalled());
+
+      exp->using_concurrency_range = true;
+      exp->concurrency_range.start = 10;
+      exp->concurrency_range.end = 10;
+      exp->max_threads = 16;
+    }
+
+    SUBCASE("Max_threads set to default when concurrency-range.end = 16")
+    {
+      args.push_back(option_name);
+      args.push_back("10:16");  // start
+
+      int argc = args.size();
+      char* argv[argc];
+      std::copy(args.begin(), args.end(), argv);
+
+      REQUIRE_NOTHROW(act = parser.Parse(argc, argv));
+      CHECK(!parser.UsageCalled());
+
+      exp->using_concurrency_range = true;
+      exp->concurrency_range.start = 10;
+      exp->concurrency_range.end = 16;
+      exp->max_threads = 16;
+    }
+
+    SUBCASE(
+        "Max_threads set to concurrency-range.end when concurrency-range.end > "
+        "16")
+    {
+      args.push_back(option_name);
+      args.push_back("10:40");  // start
```

Comment:
Is the start comment there for a reason in these test cases?

Reply:
It's an already existing comment. As far as i remember, I didn't add it. Let me recheck.

Reply:
Removed